pylutron/__init__.py

```diff
@@ -853,13 +855,91 @@ def __init__(self, lutron, name, integration_id):
     self._integration_id = integration_id
 
 
+class OccupancyGroup(LutronEntity):
+  _CMD_TYPE = 'GROUP'
+  _ACTION_STATE = 3
+
+  OCCUPIED = 3
+  VACANT = 4
+  UNKNOWN = 255
+
+  class Event(LutronEvent):
+    """OccupancyGroup event that can be generated.
+
+    OCCUPANCY_CHANGED: Occupancy state has changed.
```

Comment:
fix comment to match definition on line 873

Reply:
I don't understand this.